AuthenticatorBridgeKit/AuthenticatorBridgeItemDataModel.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+
+/// A struct for storing information about items that are shared between the Bitwarden and Authenticator apps.
+///
+public struct AuthenticatorBridgeItemDataModel: Codable, Equatable {
```

Comment:
🤔 I feel like there was some reason I didn't make the `AuthenticatorItemDataModel` to conform to `Equatable` in the Authenticator, though I don't remember why, exactly. Is there a particular reason we want it here?

(I'm also finding myself wondering, in light of the `ManagedObject` conversation, if we *actually* might want to eventually consolidate the `AuthenticatorItemDataModel` and `AuthenticatorBridgeItemDataModel` into one type that's just used by both apps; it just so happens the BWA has local storage of those objects as well 🤔 Not sure how that would play with Core Data, though. On the other hand...would it potentially be possible to just...use the `AuthenticatorItemDataModel` and move it into this library, to be shared code to begin with? And update BWA to use that instead?)

Reply:
🤔 That's an interesting question. As far as CoreData, I'm not sure how it would work to have the local data store (writing to the app's container) and the shared data store (writing to the app's app group) using the exact same CoreData model. It's possible that's fine if we just reference everything properly. I could try doing a spike?

But also, that would mean that Authenticator immediately becomes wholly dependent on the Swift package for even its own local storage. The items are pretty much identical, but I wasn't sure if we want to jump all the way in on that.

Reply:
I think it's fine currently, despite them being identical. Probably not worth digging in on that too much at the moment, but something to keep in mind in the future.